src/lsp-server.ts

```diff
@@ -299,7 +300,8 @@ export class LspServer {
         if (this.options.ctagsPath) {
             return this.options.ctagsPath;
         } else {
-            return 'ctags';
+            const packageRoot = findRoot(__filename);
```

Comment:
actually just use require.resolve would be enough, no need extra deps

Reply:
Got it.